torch/_subclasses/fake_impls.py

```diff
@@ -630,7 +630,7 @@ def multi_device_op_default(fake_mode, func, *args, **kwargs):
 @register_op_impl(aten.slice_scatter.out)
 def multi_device_op_out(fake_mode, func, *args, **kwargs):
     with in_kernel_invocation_manager(fake_mode):
-        out = func(*args, **kwargs)
+        func(*args, **kwargs)
 
     _, new_kwargs = normalize_function(
         func, args=args, kwargs=kwargs, normalize_to_only_use_kwargs=True
```

Comment:
Is the above part a bug? The function is called again right below with `normalize_function`?

Reply:
Oh wait, nroamlize function doesn't actuall call it?

Reply:
It seems `normalize function` takes a possible function call and normalizes the arguments, but doesn't call it.

Reply:
Do we actually need to call it though? That's what I am confused by